test/integration/helpers_test.go

```diff
@@ -420,14 +421,36 @@ func setIngressClassNameWithRetry(ctx context.Context, namespace string, ingress
 	})
 }
 
+type routerFlavor string
+
+const (
+	traditional           routerFlavor = "traditional"
+	traditionalCompatible routerFlavor = "traditional_compatible"
+	expressions           routerFlavor = "expressions"
+)
+
+func skipTestForRouterFlavor(t *testing.T, flavor ...routerFlavor) {
```

Comment:
This already deserves it's own file I believe ( together with other related `func`s ).

If it's not going to be used in other types of tests (other than integration tests) than it can be placed in this dir.

Reply:
yeah I agree, I'm moving it separate file, btw. I would advocate for putting all of such helpers in one folder (not as a part of this PR), because now it's not obvious at first glance if the file contains tests or some helpers